src/main/java/sirius/db/mixing/properties/MultiLanguageStringProperty.java

```diff
@@ -110,7 +110,8 @@ protected MultiLanguageString getMultiLanguageString(Object entity) {
     protected Object transformFromMongo(Value object) {
         Map<String, String> texts = new LinkedHashMap<>();
         for (Document document : (List<Document>) object.get()) {
-            texts.put(document.get(LANGUAGE_PROPERTY).toString(), document.get(TEXT_PROPERTY).toString());
```

Comment:
note that this is not the original error - as the true error is storing a null value at all - still making this code more robust is a good idea.